Source/multi.cpp

```diff
@@ -62,6 +62,7 @@ uint8_t gbDeltaSender;
 bool sgbNetInited;
 uint32_t player_state[MAX_PLRS];
 Uint32 playerInfoTimers[MAX_PLRS];
+bool isLoopback;

```

Comment:
Globals starts with capitals:
```suggestion
bool IsLoopback;
```

Preferably there would be a way to check the network type, but I'm not sure if that is possible and probably a bit hairy to get working. So if you correct this bit I think it's good enough to merge.

Reply:
Got it!